pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -50,29 +50,17 @@ def __init__(self) -> None:
         self.running_loss: TensorRunningAccum = TensorRunningAccum(window_length=20)
         self.batch_idx: int = 0
         self.split_idx: Optional[int] = None
-        self.progress = BatchProgress()
+        self.progress = Progress()
```

Comment:
@tchaton why did you change the type to the super type here?

Reply:
Reverted back to `BatchProgress `

Reply:
BatchProgress does not do anything. Can be added back if necessary later